### PR TITLE
Add Rungta College name

### PR DESCRIPTION
**Pull request by abhisekbharti**

### CODE REVIEW

**Comment by philipto** [comment]
@abhisekbharti First stage of verification passed, official domain is [rungta.ac.in](https://rungta.ac.in) as far as I verified. This is just an informational note. Pull request is still in the review. I appreciate your patience while waiting for the final decision. Proof of domain ownership: https://rcet.rungta.ac.in/arc-committees.php

**Comment by philipto** [comment]
@abhisekbharti Pull request merged. Thank you.

File: lib/domains/org/rungta.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Rungta College of Engineering and Technology
```
Comment (abhisekbharti) [inline lib/domains/org/rungta.txt:1]: This college are not verified github education global

**Comment by abhisekbharti** [comment]
https://github.com/JetBrains/swot/pull/19238/commits/05ca21a64ed15c39c9af1423cd4dc00b93bcc13e#r1659023003

**Pull request merged**